relm4-macros/tests/ui/compile-fail/bad_impl.stderr

```diff
@@ -24,11 +18,12 @@ warning: unused import: `gtk`
   |
   = note: `#[warn(unused_imports)]` on by default
 
-error[E0046]: not all trait items implemented, missing: `Root`, `init_root`
+error[E0046]: not all trait items implemented, missing: `Root`, `Widgets`, `init_root`
```

Comment:
Can we still emit `Widgets` even if the rest of the expansion will fail? See #314.

Reply:
I see. This specific case doesn't seem worth improving.